packages/workbox-core/index.mjs

```diff
@@ -22,6 +22,60 @@ class WorkboxCore {
   }
 
   /**
+   * This method returns the cache names used by Workbox. `cacheNames.precache`
+   * is used for the precached assets and `cacheNames.runtime` for everything
+   * else.
+   * @return {Object} An object with `precache` and `runtime` properies.
+   */
+  get cacheNames() {
+    return {
+      precache: _private.cacheNameProvider.getPrecacheName(),
+      runtime: _private.cacheNameProvider.getRuntimeName(),
+    };
+  }
+
+  /**
+   * You can alter the default cache names by changing
+   * the cache name details.
+   *
+   * Pass in an object containing any of the following properties,
+   * `prefix`, `suffix`, `precache` and `runtime`.
+   *
+   * Cache names are generated as `<prefix>-<precache or runtime>-<suffix>`.
+   * @param {Object} details
+   */
+  setCacheNameDetails(details) {
+    if (process.env.NODE_ENV !== 'production') {
+      Object.keys(details).forEach((key) => {
```

Comment:
Style nit, should we be favoring `for (const {key, value} of Object.entires(...)` here?

Reply:
That throws for me with `Object.entries is not a function` on node version 8.2.1 Which means it'll probably have problems in the browser.